Source/WebCore/platform/graphics/HEVCUtilities.h

```diff
@@ -59,7 +59,7 @@ WEBCORE_EXPORT std::optional<HEVCParameters> parseHEVCDecoderConfigurationRecord
 WEBCORE_EXPORT String createHEVCCodecParametersString(const HEVCParameters&);
 
 struct DoViParameters {
-    enum class Codec { AVC1, AVC3, HEV1, HVC1 } codec;
+    enum class Codec { AVC1, AVC3, HEV1, HVC1 } codec { Codec::HVC1 };
```

Comment:
Why default to the `3` value instead of the `0` value?

Reply:
I was trying to match the code above in this file:
```
struct HEVCParameters {
    enum class Codec { Hev1, Hvc1 } codec { Codec::Hvc1 };
```